osx/Keybase.xcodeproj/project.pbxproj

```diff
@@ -1399,7 +1399,7 @@
 				PROVISIONING_PROFILE_SPECIFIER = "";
 				STRIP_INSTALLED_PRODUCT = NO;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 4.0;
```

Comment:
Is that needed

Reply:
Xcode doesnt build that version anymore, it's an error.